src/modules/user/userSwagerDocs.js

```diff
@@ -1,478 +1,499 @@
+ *                       type: string
+ *                       format: email
+ *                       description: The user's email.
+ *                       example: example@example.com
+ *                     avatar:
+ *                       type: string
+ *                       format: URL
+ *                       description: Avatar URL.
+ *                       example: example.com/example.jpeg
+ *                     about:
+ *                       type: string
+ *                       description: Details about te user.
+ *                       example: "I'm a student of front-end....."
+ *                     skills:
+ *                       type: [string]
+ *                       description: Array of users skills.
+ *                       example: [HTML, CSS, JavaScript, Git, GitHub]
+ *                     applications:
+ *                       type: [object]
+ *                       description: An array of application objects
+ *                       example: [application]
+ *                     offers:
+ *                       type: [object]
+ *                       description: An array of offers objects
+ *                       example: [offer]
+ *                     listings:
+ *                       type: [object]
+ *                       description: An array of listing objects
+ *                       example: [listing]
+ *                     Created:
+ *                       type: string
+ *                       format: date
+ *                       description: Date user joined.
+ *                       example: 2023-01-16T19:55:13.609Z
+ *                     Updated:
+ *                       type: string
+ *                       format: date
+ *                       description: Most recent update.
+ *                       example: 2023-01-16T19:55:13.609Z
+ *       500:
+ *         description: Internal server error
+ *     security:
+ *       - bearerAuth: []
+ */
+
+//------------------ DELETE /users/{id} --------------------
+/**
+ * @swagger
+ * /users/{id}:
+ *   delete:
+ *     summary: delete a user.
+ *     tags: [Users]
+ *     description: Deletes a users account and all related applications and offers. User ID must match user being deleted.
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         required: true
+ *         description: String ID of the user to retrieve.
+ *         schema:
+ *           type: String
+ *     responses:
+ *       200:
+ *         description: Account deleted.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 data:
+ *                   type: object
+ *                   properties:
+ *                     message:
+ *                       type: string
+ *                       description: Success message.
+ *                       example: Account deleted.
+ *       500:
+ *         description: Internal server error

```

Comment:
The updates to delete added some 401 errors
https://github.com/NoroffFEU/agency-api.noroff.dev/blob/main/src/modules/user/controllers/controllerDelete.js

Reply:
dont think I added all of the 401`s as the errors did the same, or should I add all of them?